### DISCUSSION TITLE
How to differentiate capitated amounts for services based on factors like age, geo location, and other factors?

### DISCUSSION THREAD

**Discussion by SatyaKomatineni** [CMSgov/price-transparency-guide#482] [Q&A] [answered]
Hello,
On the capitated in network file there 

capitation id (or description): Office services
Provider 1: $40
provider 2: $60

What happens if the office services are more like:

capitation id (or description): Office services for < 18 years old for Jacksonville
Provider 1: $40
provider 2: $60

capitation id (or description): Office services for < 18 years old for NY
Provider 1: $400
provider 2: $600

Questions:

1. is this acceptable?
2. is the amount per member per month of capitation?

Appreciate it
Satya

**Comment by Kam-cog** [comment]
Note: I am not an expert but happened to read the rules in detail today and made notes for every reimbursement type thats described in the final rules - page 246 onwards.

Here is an example of reimbursement illustration that they have given which is based on a percentage which would vary based on the characteristics of Enrollee and provider. This is the closest match i could get. It states that for value based reimbursements, the base negotiated rate should suffice and the plans need not publish the other value based factors calculation .
The below is taken AS IS from the rules.
Example:a plan calculates provider reimbursement using a reference-based 
pricing model that sets reimbursement to Provider X at 120 percent of the Medicare rate for 
covered Item A. The reference-based percentage used to determine the base negotiated rate 
would be 120 percent.  In the general course of business, the plan determines the Medicare rate 
for Item A using participant, beneficiary, or enrollee-specific characteristics, but, because there is 
no specific participant, beneficiary, or enrollee for purposes of populating the In-network Rate 
File, the plan or issuer must report the base negotiated rate that would apply prior to application 
of any participant, beneficiary, or enrollee-specific characteristics. In this example, the Medicare 
rate for Item A is $150, before applying adjusters for participant, beneficiary, or enrollee-specific 
characteristics.  Therefore, the plan would report a negotiated rate for Item A when received 
from Provider X of $180 ($150 multiplied by 120 percent) and must include this rate in the In
network Rate File.

**Comment by tdfow** [comment, marked as answer]
I understood the mandate to state that if the capitation arrangement is weighted upon characteristics, such as age, gender, then the rate that is published should be before any adjustments are made---there is a reference to 'notate' any entries that represent the items/services that associate with this capitation amount---but unclear how this would be 'notated' to show it has weighted characteristics (if this should be addressed).

**Reply by ks2178o2** [reply]
Are there specific directives on how to address these cases now?
